From: Antonio Radici <antonio@debian.org>
Date: Wed, 5 Mar 2014 10:09:15 +0100
Subject: __separator__mutt.org

Dummy patch to serve as marker separating patches for mutt-patched
---
 .mutt.org | 1 +
 1 file changed, 1 insertion(+)
 create mode 100644 .mutt.org

diff --git a/.mutt.org b/.mutt.org
new file mode 100644
index 0000000..d0d32f3
--- /dev/null
+++ b/.mutt.org
@@ -0,0 +1 @@
+# dummy patch to serve as marker separating patches for mutt-patched
